resources/views/inputs/includes/input-label.blade.php

```diff
@@ -1,11 +1,13 @@
 @props([
     'name',
     'errors',
-    'id'             => null,
-    'label'          => null,
-    'tooltip'        => null,
-    'required'       => false,
-    'auxiliaryTitle' => '',
+    'id'               => null,
+    'label'            => null,
+    'tooltip'          => null,
+    'tooltipClass'     => null,
```

Comment:
do you need this class in MSQ? otherwise remove it; no need to add unused property for the "what if" case; we can add those later when they are actually needed

Reply:
I do use it within MSQ